root/release/edit/tracklist.tt

```diff
@@ -349,8 +349,16 @@
     <!-- /ko -->
 
     <div data-bind="visible: loaded() && !collapsed()">
+
+      <p>
```

Comment:
I feel like this needs `margin: 1em` 'cause it looks smooshed against the left border.